configure.ac

```diff
@@ -487,9 +471,8 @@ AC_MSG_RESULT([
 	includedir:		${includedir}
 	testdir:		${testdir}
 	WITH_ARCH:              ${WITH_ARCH}
-	with_openssl:           ${with_openssl}
 
-	cc:			${CC}
+	Cc: ${CC}
```

Comment:
Unnecessary modification

Reply:
checkpatch fails without this change.

Reply:
I'm think the checkpatch failure may be a false positive. Checkpatch confuses this line to email CC field.

Reply:
Agree with Matias that it's better to leave it as "cc:". We'll ignore the false positive error message while reviewing.